src/components/Dropdown/Dropdown.tsx

```diff
@@ -423,15 +426,20 @@ const Dropdown = ({
               // get parent before nextElementSibling because buttons are nested inside of skeletons
               const optionsContainer = button ? button.nextElementSibling : null;
               if (optionsContainer) {
-                const toFocus = optionsContainer.children[0] as HTMLElement | undefined;
+                const toFocus = optionsContainer.children[0]?.children[0]?.children[0]
+                  ?.children[0] as HTMLElement | undefined;
```

Comment:
Wha-what

Maybe we should have another variable for storing what this is. Like "firstOption" that describes what the great grandchild of the options container is